launch/launch/actions/execute_local.py

```diff
@@ -464,9 +480,35 @@ def printer(context, msg, timeout_substitutions):
             ],
             cancel_on_shutdown=False,
         )
+        def kill_subprocesses(
+            context,
+            timeout_substitutions,
+            children=psutil.Process(
+                self._subprocess_transport.get_pid()).children(recursive=True)
+        ):
+            process_name = context.locals.process_name
+            for p in children:
+                try:
+                    p.send_signal(signal.SIGKILL)
+                except psutil.NoSuchProcess:
+                    continue
+                self.__logger.warn(
```

Comment:
For the other cases (scalating SIGINT->SIGTERM and SIGTERM->SIGKILL), we're logging an error.
But I think a warning may be more appropriate, or even an INFO message.

Reply:
A warning sounds okay to me.

Reply:
+1 for a warning.